ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/LoadedPaywallComponents.kt

```diff
@@ -64,23 +74,29 @@ internal fun LoadedPaywallComponents(
     val footerComponentStyle = state.stickyFooter
     val background = rememberBackgroundStyle(state.background)
 
-    Column(modifier = modifier.background(background)) {
-        ComponentView(
-            style = style,
-            state = state,
-            onClick = clickHandler,
-            modifier = Modifier
-                .fillMaxWidth()
-                .weight(1f)
-                .verticalScroll(rememberScrollState()),
-        )
+    Box(modifier = modifier.background(background)) {
+        var footerHeight by remember { mutableIntStateOf(0) }
```

Comment:
I thought about using SubcomposeLayout for this instead but this seemed easier for this use case? wdyt?